### ISSUE TITLE
Translating Date Timeline

### ISSUE THREAD

**Issue by alexmonteiro** [daattali/timevis#40]
@daattali , nice job! This is a awesome package that helps me a lot.

I would like to know if its possible to configurate timevis() to return de description of the month in others languages. I mean that always I use tivevis() the time line of months comes like "August 2014", "September 2014"... and I would to find a way to translate it before plot. Is it possible?

**Comment by daattali** [comment]
Good question! The timevis package is simply a binding to a javascript library, so the actual logic is not done by my package. You should look at the documentation for the javascript library and see if they support other languages. If it's supported by the library, then there should be a way to use `options` in timevis to do that. Check here: visjs.org/docs/timeline/

**Comment by alexmonteiro** [comment]
@daattali ,

 Great! It works for me when I exported the timvis() graph to HTML, and there I imported moment.js.  I've just set the locale options information.

```
config <- list( locale = 'pt-br')
timevis(data, options = config)

```
Unfortunately timeline doesn't import moment-with-locales.js or moment.js files, so it would be I great improve if timevis() package import this files, so the R console or viewer would show the timeline already translated. Is it possible?
